osrdyne/src/drivers/kubernetes.rs

```diff
@@ -207,7 +220,7 @@ impl KubernetesDriver {
         };
 
         if let Some(version) = current_hpa_version {
-            info!("Updating HPA: {:?}", hpa);
+            debug!("Updating HPA: {:?}", hpa);
```

Comment:
Now that we use `tracing`, we might as well use it to send structured event logs instead? Take a look at all the modified log events of this PR, most of them are also concerned.

```suggestion
            debug!(?hpa, "Updating HPA");
```